### PR TITLE
add makeRobotAccountant solution

### PR DESCRIPTION
**Pull request by DidenkoDenis306**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,19 @@
...
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return (firstNumber) => {
+    return (secondNumber) => {
+      countCalls++;
+
+      if (countCalls <= 3 || countCalls % 2 !== 0) {
```
Comment (33Orange) [inline src/makeRobotAccountant.js:15]: The condition should be countCalls > 3 and countCalls % 2 === 0 for even calls to return an error.

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return (firstNumber) => {
+    return (secondNumber) => {
```
Comment (33Orange) [inline src/makeRobotAccountant.js:12]: Consider renaming 'secondNumber' to 'b' for brevity and consistency with the task description.

File: src/makeRobotAccountant.js [hunk 3]
```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return (firstNumber) => {
```
Comment (33Orange) [inline src/makeRobotAccountant.js:11]: Consider renaming 'firstNumber' to 'a' for brevity and consistency with the task description.

File: src/makeRobotAccountant.js [hunk 4]
```diff
@@ -6,7 +6,19 @@
...
+    return (secondNumber) => {
+      countCalls++;
+
+      if (countCalls <= 3 || countCalls % 2 !== 0) {
+        return firstNumber + secondNumber;
```
Comment (33Orange) [inline src/makeRobotAccountant.js:16]: Returning correct sum for odd calls should be outside of this if statement, move it to the else block.

**Review by 33Orange — APPROVED** [review]
Well done ! 🔥

**Review by dinaminahi — APPROVED** [review]

**Pull request closed without merging**